main.py

```diff
@@ -46,20 +48,12 @@ class game:
 
     def make(self):
```

Comment:
We would normally call this method `run`, as that would better fit the usage in terms of Sugargame.

Reply:
done